### PR TITLE
Link to each component's README#dependencies

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/third-party/_index.md [hunk 1]
```diff
@@ -10,19 +10,17 @@ K8ssandra provides a production-ready platform for running Apache Cassandra&reg;
...
-|yaml                                                                |9fc95527decd95bb9d28cc2eab08179b2d0f6971                                   |Multi-license: BSD-3-Clause OR MIT; BSD-3-Clause                                                                                                                                                                                                                                                                                                                                 |
+For each component deployed by K8ssandra, see the following 3rd-party license reports. Each one was generated by the Fossa app. Start in the #dependencies section of each component's GitHub-hosted README. From there, access the latest generated "open source report" link.
```
Comment (bradfordcp) [inline docs/content/en/third-party/_index.md:13]: Link to Fossa?

File: docs/content/en/third-party/_index.md [hunk 2]
```diff
@@ -10,19 +10,17 @@ K8ssandra provides a production-ready platform for running Apache Cassandra&reg;
...
+
+* [K8ssandra dependencies]({{< relref "https://github.com/k8ssandra/k8ssandra#dependencies" >}}) 
+* [Cass Operator dependencies]({{< relref "https://github.com/k8ssandra/cass-operator/#dependencies" >}})
+* [Medusa Operator dependencies]({{< relref "https://github.com/k8ssandra/medusa-operator/#dependencies" >}})
+* [Reaper Operator dependencies]({{< relref "https://github.com/k8ssandra/reaper-operator/#dependencies" >}})
+* [Management API for Apache Cassandra dependencies]({{< relref "https://github.com/k8ssandra/management-api-for-apache-cassandra/#dependencies" >}})
+* [Go client library for the Reaper for Apache Cassandra REST API dependencies]({{< relref "https://github.com/k8ssandra/reaper-client-go/#dependencies" >}})
+* [Cassandra Reaper dependencies]({{< relref "https://github.com/thelastpickle/cassandra-reaper/#dependencies" >}})
+* [Cassandra Medusa dependencies]({{< relref "https://github.com/thelastpickle/cassandra-medusa/#dependencies" >}})
```
Comment (bradfordcp) [inline docs/content/en/third-party/_index.md:22]: I would remove "dependencies" from each of the links.

File: docs/content/en/third-party/_index.md [hunk 3]
```diff
@@ -32,4 +30,4 @@ The following table lists the K8ssandra direct dependencies with 3rd-party softw
 * [Tasks]({{< relref "tasks" >}}): Need to get something done? Check out the Tasks topics for a helpful collection of outcome-based solutions.
 * [Reference]({{< relref "reference" >}}): Explore the K8ssandra configuration interface (Helm charts), the available options, and a Glossary.
 
-We encourage you to actively participate in the [K8ssandra community](https://k8ssandra.io/community/).
\ No newline at end of file
+Also, we encourage you to actively participate in the [K8ssandra community](https://k8ssandra.io/community/).
```
Comment (bradfordcp) [inline docs/content/en/third-party/_index.md:33]:
```suggestion
Join us as part of the [K8ssandra community](https://k8ssandra.io/community/)!
```

**Review by bradfordcp — CHANGES_REQUESTED** [review]

File: docs/content/en/third-party/_index.md (hunk 1 above)
Comment (bradfordcp) [inline docs/content/en/third-party/_index.md:13]:
```suggestion
For each component deployed by K8ssandra, see the following 3rd-party license reports. Each one was generated by [Fossa](https://fossa.com/). See the "Dependencies" section of each component's GitHub README. From there, access the latest generated "open source report" link.
```

File: docs/content/en/third-party/_index.md (hunk 2 above)
Comment (bradfordcp) [reply docs/content/en/third-party/_index.md:22]:
```suggestion
* [K8ssandra]({{< relref "https://github.com/k8ssandra/k8ssandra#dependencies" >}}) 
* [Cass Operator]({{< relref "https://github.com/k8ssandra/cass-operator/#dependencies" >}})
* [Medusa Operator]({{< relref "https://github.com/k8ssandra/medusa-operator/#dependencies" >}})
* [Reaper Operator]({{< relref "https://github.com/k8ssandra/reaper-operator/#dependencies" >}})
* [Management API for Apache Cassandra]({{< relref "https://github.com/k8ssandra/management-api-for-apache-cassandra/#dependencies" >}})
* [Go client library for the Reaper for Apache Cassandra REST API]({{< relref "https://github.com/k8ssandra/reaper-client-go/#dependencies" >}})
* [Cassandra Reaper]({{< relref "https://github.com/thelastpickle/cassandra-reaper/#dependencies" >}})
* [Cassandra Medusa]({{< relref "https://github.com/thelastpickle/cassandra-medusa/#dependencies" >}})
```

File: docs/content/en/third-party/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/third-party/_index.md:13]: Good idea.

File: docs/content/en/third-party/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/third-party/_index.md:22]: Will do.

File: docs/content/en/third-party/_index.md (hunk 3 above)
Comment (johnsmartco) [reply docs/content/en/third-party/_index.md:33]: Nice.

**Review by bradfordcp — APPROVED** [review]

**Pull request merged**